src/robot/parsing/populators.py

```diff
@@ -28,11 +28,11 @@
 from .tsvreader import TsvReader
 from .txtreader import TxtReader
 from .restreader import RestReader
-
+from .mdreader import MarkDownReader
 
 READERS = {'html': HtmlReader, 'htm': HtmlReader, 'xhtml': HtmlReader,
-           'tsv': TsvReader , 'rst': RestReader, 'rest': RestReader,
```

Comment:
Fixing small PEP-8 errors here in the code that you edit is fine, but in general it would be better to have them in separate PRs. Having lot of unrelated changes in a PR makes reviewing it harder.